src/app/pages/MainArticles/main.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { PureComponent } from 'react';
+import { Link } from 'react-router-dom';
+import { InjectedIntlProps } from 'react-intl';
+import { v1 } from 'uuid';
+import dayjs from 'dayjs';
+import { defaultTheme } from '@core/styles/themes';
+
+import {
+  AppWrapper,
+  ContentWrapper,
+  ArticleDate,
+  ArticleAuthor,
+  ArticleTitle,
+  ArticleDescription,
+  ArticleImage,
+  Title,
+  TitleWrapper,
+} from '@core/components';
+import {
+  StyledButton,
+  ArticleButton,
+  ArticleButtonText,
+} from '@core/components/button';
+import {
+  ArticlesWrapper,
+  ArticleWrapper,
+  FiltersWrapper,
+  FilterWrapper,
+  FilterButtonWrapper,
+  Select,
+} from './components';
+
+interface OwnProps {
+  fetchArticles: any;
+  filterArticles: any;
+  clearArticlesFilters: any;
+  articles: any;
+  topic: string;
+  sortBy: string;
+  date: any;
+  handleDate: any;
+}
+const image = require('../../components/image/images/logo_espeo.svg');
+
+type MainArticlesProps = OwnProps & InjectedIntlProps;
+
+class MainArticles extends PureComponent<MainArticlesProps> {
+  public componentDidMount() {
+    const { fetchArticles } = this.props;
+    fetchArticles(1, { topic: '', sortBy: '', date: '' });
+  }
+
+  changeTitle = (title: string) => {
```

Comment:
since you declared access modifier for this method, why not for the rest of them?